### PR TITLE
Update starterkit

### PR DESCRIPTION
**Pull request by eTorAken**
- updated starterkit
- fixed some eslint warnings
- updated deps

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ import { extractStyles } from 'react-statics-styles';
 
 const PLUGIN_NAME = 'gulp-react-statics-styles';
 
-module.exports = function() {
-  return through.obj(function(file, enc, fn) {
+function Wrapper() {
```
Comment (eTorAken) [inline src/index.js]: I don't know if this was the name you would have put there

Comment (elierotenberg) [inline src/index.js]:
```
export default () => {
  ..
}
```

File: dist/index.js [hunk 2]
```diff
@@ -1,62 +1,69 @@
...
-      var moduleFile = join(base, relative);
+      var moduleFile = _join.join(base, relative);
       var moduleName = require.resolve(moduleFile);
       if (require.cache[moduleName] !== void 0) {
         delete require.cache[moduleName];
       }
       var Component = require(moduleFile);
-      var styles = extractStyles(Component);
+      var styles = _extractStyles.extractStyles(Component);
       var contents = undefined;
```
Comment (eTorAken) [inline dist/index.js:53]: expected? Buffer seems undeclared

File: dist/index.js [hunk 3]
```diff
@@ -1,62 +1,69 @@
...
       try {
         contents = new Buffer(styles);
       } catch (err) {
         return fn(null);
       }
-      path = replaceExtension(path, ".css");
-      this.push(new File({ path: path, contents: contents }));
+      path = _PluginError$File$replaceExtension.replaceExtension(path, '.css');
+      undefined.push(new _PluginError$File$replaceExtension.File({ path: path, contents: contents }));
```
Comment (eTorAken) [inline dist/index.js]: same

**Pull request merged**